### DISCUSSION TITLE
Does audio downloaded as wav have better quality than audio downloaded as mp3?

### DISCUSSION THREAD

**Discussion by ChocolateLoverRaj** [aandrew-me/ytDownloader#147] [Q&A]
I want to download the best possible quality, but if wav isn't better then mp3 then I won't unnecessarily have larger files.
![image](https://github.com/aandrew-me/ytDownloader/assets/52586855/567877d1-0cbc-45ad-8b69-7b881cc7b626)

**Comment by aandrew-me** [comment]
Since conversion is taking place here, I don't think there is a big difference. I suggest you download both and just compare and see
